lib/core.json

```diff
@@ -15,6 +15,7 @@
     "events": true,
     "freelist": "< 6",
     "fs": true,
+    "fs/promises": ">= 10.0.0",
```

Comment:
hm, this seems like it would conflict with `node_modules/fs/promises.js`, for example